### ISSUE TITLE
Where all `mocks.py` functions should go?

### ISSUE THREAD

**Issue by bazkiaei** [AstroHuntsman/huntsman-mocks#78]
@bazkiaei I like this definition:

> Looks to me like /utils is a place where you can place small snippets you can use throughout the application. Small functions to build bigger things with.

from https://github.com/erikras/react-redux-universal-hot-example/issues/808 

You designed `parse_config` to be used specifically for `mocks.py`, so I guess it makes sense. 

This is scope creep on the PR, so let's close this PR without changing this, but create a new Issue to decide where *all* `mocks.py` functions should go -- other ones I think meet the definition I copied in above. Or better yet add it to the discussion about creating a Class for Mocks: #63

_Originally posted by @lspitler in https://github.com/AstroHuntsman/huntsman-mocks/pull/69_

**Comment by lspitler** [comment]
Discuss after #71  and @wtgee and @AnthonyHorton about advise on how to structure
